Orso.Arpa.Persistence/Migrations/20210619135559_UpdateLocalizationTable.Designer.cs

```diff
@@ -794,7 +794,7 @@ protected override void BuildTargetModel(ModelBuilder modelBuilder)
                             CreatedAt = new DateTime(2021, 6, 16, 15, 30, 19, 324, DateTimeKind.Local).AddTicks(7866),
                             CreatedBy = "LocalizationSeedData",
                             Deleted = false,
-                            Key = "Invalid token supplied",
```

Comment:
Migration files are automatically generated by entity framework and must not be updated manually - they are already executed on the prod database!
Please update the files in the "Translation" folder and execute a new migration instead (see how-to in readme file of the persistence project)